golem/managers/proposal/plugins/buffer.py

```diff
@@ -1,130 +1,103 @@
-import asyncio
 import logging
-from asyncio import Queue
-from typing import List
 
 from golem.managers import ProposalManagerPlugin
 from golem.resources import Proposal
-from golem.utils.asyncio import create_task_with_logging
-from golem.utils.logging import get_trace_id_name, trace_span
+from golem.utils.asyncio.buffer import BackgroundFillBuffer, SimpleBuffer
+from golem.utils.logging import trace_span
 
 logger = logging.getLogger(__name__)
 
 
-class Buffer(ProposalManagerPlugin):
+class BufferPlugin(ProposalManagerPlugin):
```

Comment:
shouldn't the name here be `ProposalBufferPlugin`? what if we have a `BufferPlugin` in some other manager?

Reply:
You touched quite interesting topic that have some pros and cons - I'd like to talk it through on some meeting.